doc/scenario.md

```diff
@@ -291,7 +291,7 @@ List of facts:
   - `name` - instance name (Ansible host);
   - `console_sock` - path to control socket of instance;
 - `cartridge_scenarios` - finally dictionary with scenarios (combination of role and user scenarios);
-- `cartridge_scenario` - steps of selected scenario (if not previously defined);
+- `scenario_steps_names` - names of scenario steps;
```

Comment:
Variable `scenarios` was lost

Reply:
It was `cartridge_scenarios`. I've renamed it.